doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -331,6 +331,15 @@ Field              | Type           | Meaning
 `id`               | [`text`]       | The ID of the source. Corresponds to [`mz_catalog.mz_sources.id`](../mz_catalog#mz_sources).
 `replication_slot` | [`text`]       | The name of the replication slot in the PostgreSQL database that Materialize will create and stream data from.
 
+### `mz_sessions`
```

Comment:
I couldn't figure out any discernible order to the tables/views in this file, but this seemed like a good spot.

Reply:
It was originally alphabetical, but definitely is not anymore. Other option would be before `mz_subscriptions` since that relation will join against `mz_sessions`. Either one works for me.

Reply:
That sounds like a better spot, just moved it there.